docs/install-1.7.md

```diff
@@ -21,65 +32,75 @@ kubernetes infrastructure CAs or certificates. This script should be
-hold the generated config and certificates, before running the script.
+Please create a fresh directory called `certs` to hold the configuration
+and certificates that we'll generate in the following steps:
 
-## Check that aggregator is enabled
+```console
+mkdir certs
+```
 
-`kubectl api-versions` MUST list `apiregistration.k8s.io/v1beta1` (or
-`apiregistration.k8s.io/v1alpha1` if running in kubernetes v1.6)
+## Check That the API Aggregator is Enabled
 
-This API Group will not show up if you are talking to the insecure
-port. The insecure port is not behind the aggregator. The aggregator
-does not support routing of requests to the insecure port. You must
-talk to the secure port, as the aggregator does not serve an API on
-the insecure port.
+Run the following:
 
-## Variables used during the rest of the steps
+```console
+kubectl api-versions
+```
 
-These will be used during the steps and for the final `helm
-install`. They are key to set, as the signed certificates will be
-bound to the exact service name that is defined below as
-`SVCCAT_SERVICE_NAME`. `SVCCAT_SERVICE_NAME` will be the exact DNS
-entry that the generated certificate is bound to, so any deviation
-from the use of these defined variables will result in a certificate
-that is useless for the purposes of aggregation. All of the DNS
-entries must match.
+This endpoint must list `apiregistration.k8s.io/v1beta1`. If it does not, your 
+Kubernetes installation is likely version 1.6 or previous. We recommend running
+1.7 or later, but if you decide to continue, please see the 
+[installation document for Kubernetes 1.6](./install-1.6.md).
+
+This `apiregistration.k8s.io/v1beta1` API Group will not show up if you are 
+connecting to the insecure port of the core API server, so be sure that your
+`kubectl` configuration file (often located at `~/.kube/config`) is pointing
+to an https endpoint.
+
+## Create Environment Variables
+
+The following environment variables will be used during the following steps.
+They will be passed to the certificate generation tools as well as the final
+`helm install` step.
+
+They are important to set, as the signed certificates will be bound to the 
+exact service name that is defined below as `SVCCAT_SERVICE_NAME`. 
+
+`SVCCAT_SERVICE_NAME` will be the exact DNS entry that the generated 
+certificate is bound to, so any deviation from the use of these defined 
+variables will result in a certificate that is useless for the purposes of 
+aggregation. All of the DNS entries must match.
 
 ```
 export HELM_RELEASE_NAME=catalog
 export SVCCAT_NAMESPACE=catalog
 export SVCCAT_SERVICE_NAME=${HELM_RELEASE_NAME}-catalog-apiserver
 ```
 
-## Set up the certificate bundle
+## Create Our Own CA and Generate Keys
 
-The APIService expects a certificate bundle. We can create our own, or
+The `APIService` resource expects a certificate bundle. We can create our own, or
 pull the one from kube core for reuse.
```

Comment:
"...from the core Kubernetes API Server..."

Reply:
done